### DISCUSSION TITLE
Cannot use calcGBPD function

### DISCUSSION THREAD

**Discussion by sko1109** [mtex-toolbox/mtex#2011] [Ask Anything]
Hi everyone, 

I was trying to use calcGBPD function to plot out grain boundary plane distribution figures, following this example: 
https://mtex-toolbox.github.io/BoundaryNormalDistribution.html

Here is my code:
<img width="506" alt="Screenshot 2023-12-09 at 2 34 04 PM" src="https://github.com/mtex-toolbox/mtex/assets/63637937/05ae65f9-b3b0-4213-b256-aef2210d2096">

It says "Undefined function 'calcGBPD' for input arguments of type 'grainBoundary'." However, it is following the code example and documentation of calcGBPD

https://mtex-toolbox.github.io/grainBoundary.calcGBPD.html

Does anyone encounter the same issue and know how to solve it? I am using MTEX 5.7.0 version.

Thanks,
Shu-Ting

**Comment by kilir** [comment]
Hi,
the function does not exist in mtex 5.7, you have to use a recent version of mtex.
Cheers,
Rüdiger
